packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -346,19 +360,38 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       await this.closeDownloadingChannels(torrent);
     });
 
-    torrent.on(TorrentEvents.ERROR, err => {
-      log.error({err, torrent}, 'Torrent ERROR: ');
-      this.emit(ClientEvents.TORRENT_ERROR, {torrent, err});
+    torrent.on(TorrentEvents.ERROR, error => {
+      log.error('Torrent ERROR: ', torrent, error);
+      if (error && (error as Error).message.includes('Cannot add duplicate torrent')) {
```

Comment:
yeah, this is a ~horrible~ temporal thing to do, to lay the groundwork for pausing. This was the easiest way to avoid having more buttons to pause and resume.
The idea is to merge this and immediately after start working on the pause button. But ensuring that you (and the e2e tests, and whatever else) can start using and looking at this.
I'll see if adding this the codebase tomorrow in a better approach is easy to do so that I don't even leave this.